webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/techniqueLibraryManagement.html

```diff
@@ -1,7 +1,7 @@
 <lift:surround with="common-layout" at="content">
 
 <head>
-<title>Rudder - Technique Library Management</title>
+<title>Rudder - Technique library management</title>
```

Comment:
Can we just remove `Rudder - ` ?

Reply:
In fact it seems that we are adding `Rudder - ` in `<title>` so it's ok I guess

Reply:
I think we have it on other pages in general...